assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.appgateway_diagnostics.json

```diff
@@ -0,0 +1,30 @@
+{
+    "properties": {
+        "metadata": {
+            "assignedBy": "PlatOps"
+        },
+        "description": "Forwards App Gateway Logs to an Azure Event Hub",
+        "displayName": "HMCTS Collect App Gateway Logs",
+        "enforcementMode": "Default",
+        "nonComplianceMessages": [],
+        "notScopes": [],
+        "parameters": {
+            "eventHubAuthRule": {
+                "value": "/subscriptions/2307d175-7e49-434b-9ac2-515529b845f2/resourceGroups/soc-core-infra-sbox-rg/providers/Microsoft.EventHub/namespaces/soc-sbox-eventhubns/authorizationRules/RootManageSharedAccessKey"
+            }
+        },
+        "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/dts002/providers/Microsoft.Authorization/policyDefinitions/HMCTSAppGatewayDiagnostic",
+        "scope": "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb"
+    },
+    "location": "uksouth",
+    "identity": {
+        "type": "SystemAssigned"
```

Comment:
Use the user assigned managed identity from the keyvault policy assignment

Reply:
it's just a test assignment so not much point?

Reply:
Suppose so. If you're not going to test the remediation then there's not much point with this assignment being here at all. It's a carbon copy of the keyvault one which has been tested already